README.md

```diff
@@ -1,9 +1,346 @@
-OpenMRS Module Appointments Backend
-=================================
+# OpenMRS Module Appointments Backend
+
 This repository acts as the back end for the **Bahmni Appointment Scheduling**.
 
 ## Packaging
-```mvn clean package```
+
+`mvn clean package`
 
 The output is the OMOD file:
-```openmrs-module-appointments/omod/target/appointments-[VERSION].omod```
+`openmrs-module-appointments/omod/target/appointments-[VERSION].omod`
+
+# Setup instructions
```

Comment:
The following is not universal for all OpenMRS distro, including Bahmni. I would prefer we describe something like 

If you already have "openmrs-appointment-scheduling" module installed, then there are a few things that you must do one of the following
1. Uninstall the "openmrs-appointment-scheduling" module from the openmrs admin page
2. Identify openmrs modules directory, and remove the module file from there 
> sudo rm appointmentscheduling-1.13.0.omod

Reply:
FWIW, the module in question is `appointmentscheduling` or `openmrs-module-appointmentscheduling`, not `openmrs-appointment-scheduling`